.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,19 @@
+name: Deploy to GitHub Pages.
```

Comment:
gh-repository doesn`t have separate gh-page for each branch. Is this is a neccessary to deploy our branch?